guides/common/modules/proc_preparing-the-source-server-for-cloning.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="preparing-the-source-server-for-cloning"]
+= Preparing the source server for cloning
+
+On the source server, complete the following steps:
```

Comment:
I suggest to slightly rephrase this. IMO introductions do not end with a trailing colon. Maybe "Prepare your source {ProjectServer} for cloning." or similar.

I'd definitely drop "the following steps" because I consider it unnecessary fluff.